src/activities/share/resource/board/board0.qml

```diff
@@ -0,0 +1,47 @@
+/* GCompris
+ *
+ * Copyright (C) 2016 Stefan Toncu <stefan.toncu@cti.pub.ro>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.0
+
+QtObject {
```

Comment:
Of course it is possible. I will try to do it, but also keep in mind that the first levels should have a smaller difficulty.